### PR TITLE
SCS Wallaby Update Experience

### PR DESCRIPTION
**Pull request by matfechner**
Signed-off-by: Mathias Fechner <mathias.fechner@plusserver.com>

some little post for the  Rabbitmq User Story

### CODE REVIEW

**Comment by matfechner** [comment]
Hi  @itrich here is  my post

File: _i18n/en/_posts/blog/2022-02-01-wallaby-upgrade.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+layout: post
+title:  "Sovereign Cloud Stack Community Plusserver Environment"
+author: "Mathias Fechner"
+---
+
+_(By Markus Lindenblatt, Toens Bueker, Mathias Fechner)_
+
+### One Wallaby Update Experience
+
+Since more then one year Plusserver has provided the SCS Community Community Environment.
```
Comment (garloff) [inline _i18n/en/_posts/blog/2022-02-01-wallaby-upgrade.md:11]: double Community

File: _i18n/en/_posts/blog/2022-02-01-wallaby-upgrade.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+For the Plusserver SCS Community Environment we started our OpenStack Wallaby upgrade
+in early December 2021, everything looked good and fine.
+
+One cosmetic issue was, that the neutron-metadata-agent was missing in "openstack network agent list"
```
Comment (garloff) [inline _i18n/en/_posts/blog/2022-02-01-wallaby-upgrade.md:15]: no comma in English

File: _i18n/en/_posts/blog/2022-02-01-wallaby-upgrade.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+despite the agents being up and running.
+
+After one week we detected that some OpenStack services like octavia, nova and cinder became non-responsive.
+The reason could be traced back to the rabbitmq cluster, which was strangely running in a split brain fashion.
+After cleaning up the rabbitmq cluster and re-deploying it, everything went back to normal again.
+But it didn't take too long and services became non-responsive again... 
+After an intensive diagnostic session we decided to roll rabbitmq back to the victoria container image version,
```
Comment (garloff) [inline _i18n/en/_posts/blog/2022-02-01-wallaby-upgrade.md:22]: Would it be good to mention the rabbitmq version numbers included in the Victoria and Wallaby containers?

File: _i18n/en/_posts/blog/2022-02-01-wallaby-upgrade.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+But it didn't take too long and services became non-responsive again... 
+After an intensive diagnostic session we decided to roll rabbitmq back to the victoria container image version, 
+which helped to work around this bug.
+
+### what was the Problem?
+
+We suspect https://github.com/rabbitmq/osiris/issues/53 being the culprit, which shows some issues with split brain situations
+in Docker setups.
+
+We have filed an issue against the osism testbed in order to see, when the bug is fixed (https://github.com/osism/testbed/issues/978).
```
Comment (garloff) [inline _i18n/en/_posts/blog/2022-02-01-wallaby-upgrade.md:30]: Do we have further plans beyond reporting? E.g. working with the upstream community to deploy potential fixes and validate whether they work?

File: _i18n/en/_posts/blog/2022-02-01-wallaby-upgrade.md (hunk 3 above)
Comment (fkr) [reply _i18n/en/_posts/blog/2022-02-01-wallaby-upgrade.md:22]: @garloff makes sense to include those imho

File: _i18n/en/_posts/blog/2022-02-01-wallaby-upgrade.md (hunk 4 above)
Comment (garloff) [reply _i18n/en/_posts/blog/2022-02-01-wallaby-upgrade.md:30]: Taking this one step further: This should hit all Wallaby kolla-ansible deployments, no?

Comment (matfechner) [reply _i18n/en/_posts/blog/2022-02-01-wallaby-upgrade.md:30]: @garloff, @fkr   that is necessary to prove, but in long term running osism testbed with wallaby, we have not noticed.  The rabbitmq in the last stable versions is still fixed.  i will continue the Markdown with the planned update procedure for the rabbitmq cluster, Thank you for feedback

**Review by itrich — CHANGES_REQUESTED** [review]

**Comment by matfechner** [comment]
Hi @itrich, i hope now  it is looking better  https://github.com/SovereignCloudStack/website/pull/233/commits/545894ae837d05bd179f1a1174df5a2db87a41ca

**Review by itrich — APPROVED** [review]

**Pull request merged**